metricbeat/metricbeat.full.yml

```diff
@@ -126,6 +126,11 @@ metricbeat.modules:
   kernel.rate_limit: 0
   kernel.include_raw_message: false
   kernel.include_warnings: false
+  kernel.audit_rules: |
+    # Define audit rules here.
+    # Create file watches (-a) or syscall audits (-a or -A). For example:
```

Comment:
It was suppose to refer to them both, but has a typo. It should say "Create file watches (-w)...". Will fix.